### PR TITLE
Fix whats-new for 2023.02.0

### PR DESCRIPTION
**Pull request by dcherian**
Oops. I used "Github Codespaces" to edit whats-new, but it turns if you commit in there, it just commits to main!

This fixes the pre-commit error.

### CODE REVIEW

File: doc/whats-new.rst [hunk 1]
```diff
@@ -20,9 +20,9 @@ What's New
 v2023.02.0 (Feb 7. 2023)
```
Comment (dcherian) [inline doc/whats-new.rst:20]:
```suggestion
v2023.02.0 (Feb 7, 2023)
```

**Comment by TomNicholas** [comment]
Wait what version are we on right now?? :sweat_smile: Are you in middle of issuing a release? The `whats-new` in `main` doesn't match what's actually been released atm

**Comment by dcherian** [comment]
Sorry  for the confusion. Releasing 2023.02.0 now. I  meant to create a PR with the whats-new updates, but committed to main by mistake.

**Comment by TomNicholas** [comment]
Thanks @dcherian!

**Pull request merged**